index.js

```diff
@@ -88,7 +88,7 @@ function setActionOutput(releaseResponse) {
   const {
     data: { id: releaseId, html_url: htmlUrl, upload_url: uploadUrl }
   } = releaseResponse
-  core.setOutput('id', releaseId)
+  core.setOutput('id', releaseId.toString())

```

Comment:
😱 Cannot believe the tests did not catch this!
This was not caught because we did not return a reply to the release drafting post 😭

Caught it on the action lucky and is now caught by tests.

```
    TypeError: (s || "").replace is not a function

      89 |     data: { id: releaseId, html_url: htmlUrl, upload_url: uploadUrl }
      90 |   } = releaseResponse
    > 91 |   core.setOutput('id', releaseId)
         |        ^
      92 |   core.setOutput('html_url', htmlUrl)
      93 |   core.setOutput('upload_url', uploadUrl)
      94 | }

      at escapeData (node_modules/@actions/core/src/command.ts:78:6)
      at Command.toString (node_modules/@actions/core/src/command.ts:71:31)
      at Object.issueCommand (node_modules/@actions/core/src/command.ts:25:28)
      at Object.setOutput (node_modules/@actions/core/src/core.ts:84:3)
      at setOutput (index.js:91:8)
      at setActionOutput (index.js:83:5)
```

of course once could argue that escapeData should handle escaping numbers...